test/test_other.py

```diff
@@ -337,20 +337,19 @@ def test_emcc_output_mjs(self, args):
   })
   @node_pthreads
   def test_emcc_output_worker_mjs(self, args):
-    create_file('extern-post.js', 'await Module();')
     os.mkdir('subdir')
     self.run_process([EMCC, '-o', 'subdir/hello_world.mjs',
                       '-sEXIT_RUNTIME', '-sPROXY_TO_PTHREAD', '-pthread', '-O1',
-                      '--extern-post-js', 'extern-post.js',
                       test_file('hello_world.c')] + args)
     src = read_file('subdir/hello_world.mjs')
+    create_file('subdir/run.mjs', '''
+      import Module from "./hello_world.mjs";
+      await Module();
+    ''')
```

Comment:
Why did this change?

Reply:
This is the same reason as above.  The old code was doing `await Module()` in a post-js file, which is then executed on each thread.

Reply:
Reverted